bin/checkout.js

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+
+// node dependencies
+var fs = require('fs');
+
+// external dependencies
+var shell = require('shelljs');
+
+// parse args
+var argv = require('optimist')
+    .usage('Usage: $0 --config [path] --project [name]')
+    .demand('config')
+    .argv;
+
+function cloneProject(projectName, projectsConfig) {
+
+    var project  = projectsConfig[projectName];
+    var codebase = project.codebases[project.codebase];
+    var command  = 'git clone ' + codebase.repo + ' --branch=' + codebase.branch + ' --depth 1'
+
+    shell.exec(command, {async: true, silent: false, fatal: false});
```

Comment:
Also, we are queuing an async operation without waiting for it - I'm not sure if that's a good idea.